src/k8s/pkg/client/kubernetes/node.go

```diff
@@ -4,16 +4,17 @@ import (
 	"context"
 	"fmt"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/util/retry"
 )
 
 // DeleteNode will remove a node from the kubernetes cluster.
 // DeleteNode will retry if there is a conflict on the resource.
+// DeleteNode will not fail if the node does not
```

Comment:
nit: comment incomplete

Reply:
found a couple more things like this, will round it with a separate fix typos PR, to avoid unnecessary conflicts across #518, #520, #521